src/bootstrap-table.js

```diff
@@ -1742,16 +1742,20 @@
     BootstrapTable.prototype.initServer = function (silent, query) {
         var that = this,
             data = {},
-            params = {
-                pageSize: this.options.pageSize === this.options.formatAllRows() ?
-                    this.options.totalRows : this.options.pageSize,
-                pageNumber: this.options.pageNumber,
+            params = {                
                 searchText: this.searchText,
                 sortName: this.options.sortName,
                 sortOrder: this.options.sortOrder
             },
             request;
 
+        if(this.options.pagination)
```

Comment:
@wenzhixin - strange tab was there if just using online editor, but redid indents again and moved '{' onto same line